README.md

```diff
@@ -1,2 +1,53 @@
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+[![CircleCI](https://circleci.com/gh/py-why/dodiscover/tree/main.svg?style=svg)](https://circleci.com/gh/py-why/dodiscover/tree/main)
+[![unit-tests](https://github.com/py-why/dodiscover/actions/workflows/main.yml/badge.svg)](https://github.com/py-why/dodiscover/actions/workflows/main.yml)
+[![Checked with mypy](http://www.mypy-lang.org/static/mypy_badge.svg)](http://mypy-lang.org/)
+[![codecov](https://codecov.io/gh/py-why/dodiscover/branch/main/graph/badge.svg?token=H1reh7Qwf4)](https://codecov.io/gh/py-why/dodiscover)
+
 # dodiscover
-A Causal Discovery API for Python
\ No newline at end of file
+
+dodiscover is a Python library for causal discovery, or structure learning. This is generally considered a "first step" in the causal inference pipeline, if one does not have access to a hypothesized causal graph for their situation.
```

Comment:
Assuming "structure learning" is a bigger scope than "causal discovery" Let's narrow scope down to "Python library for causal discovery (causal structure learning)."

Reply:
Suggested rephrasing

> dodiscovery is a Python library for causal discovery (causal structure learning).  Causal discovery is useful when you aren't sure what specific causal questions you have in your domain and want to start by exploring causal relationships in the data.  dodiscovery also supports finding the causal structure you need to target specific causal inferences (e.g., cause effect inference).

I think it is best to avoid framing global causal discovery necessarily as the first step of a cause inference pipeline.  Causal discovery tends to focus on getting the "right" graph given the data you have, and most of causal effect inference methods focus on getting the "right" causal effect in a way that is robust to not knowing all of the graph or observing all of the data.  I think we'll address this with learning ADMG's, providing Markov blanket discovery, etc.  But we shouldn't overpromise.

Reply:
Done.